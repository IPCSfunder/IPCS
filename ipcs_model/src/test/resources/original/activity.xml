<?xml version='1.0' encoding='UTF-8'?>
<dataset>
	<PERSON person_objid="1" account_name="JamesChen" password_hash="12345"/>
	<PERSON person_objid="2" account_name="WilliamWang"  password_hash="12345"/>
	<PERSON person_objid="3" account_name="JohnasonChen" password_hash="12345"/>
	<PERSON person_objid="4" account_name="MichaelMiao" password_hash="12345"/>
	<PERSON person_objid="5" account_name="LuisChen" password_hash="12345"/>
	<school_type SCHOOL_TYPE_OBJID="1" NAME ="HIGH_SCHOOL"/>
	<school SCHOOL_OBJID="1" name = "PUNGOL_PRIMARY_SCHOOL" ADDRESS="PUNGOL" SCHOOL_TYPE_FK="1"/>
	<activity activity_objid="1" host="2" name="Math" location="NANNING" description="Math" start_time="2015-05-14 05:00:00" school_fk="1"></activity>
	<schedule></schedule>
</dataset>