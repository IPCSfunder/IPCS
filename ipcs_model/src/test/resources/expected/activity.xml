<?xml version='1.0' encoding='UTF-8'?>
<dataset>
	<person person_objid="1" account_name="JamesChen" password_hash="12345"/>
	<person person_objid="2" account_name="WilliamWang"  password_hash="12345"/>
	<person person_objid="3" account_name="JohnasonChen" password_hash="12345"/>
	<person person_objid="4" account_name="MichaelMiao" password_hash="12345"/>
	<person person_objid="5" account_name="LuisChen" password_hash="12345"/>
	<activity name="Math" host="2" location="NANNING" description="Math" start_time="2015-05-14 05:00:00" school_fk="1"></activity>
	<activity name="Language" host="1" location="PUNGGOL" description="Language" start_time="2015-05-12 05:00:00" school_fk="1"></activity>
	<schedule person_fk="2"></schedule>
	<schedule person_fk="4"></schedule>
	<school_type SCHOOL_TYPE_OBJID="1" NAME ="HIGH_SCHOOL"/>
	<school SCHOOL_OBJID="1" name = "PUNGOL_PRIMARY_SCHOOL" ADDRESS="PUNGOL" SCHOOL_TYPE_FK="1"/>
</dataset>